lib/pages/home/home_page.dart

```diff
@@ -91,10 +93,22 @@ class HomePage extends StatelessWidget {
                   .map(
                     (q) => ListTile(
                       title: Text(q.question),
-                      subtitle: AutoSizeText(
-                        q.answer,
+                      subtitle: AutoSizeMarkdownBody(
+                        fitContent: false,
+                        /*
+                        This is a workaround because the strings in Firebase represent
+                        newlines as '\n' and Firebase replaces them with '\\n'. We need
+                        to replace them back for them to display properly.
+                        (See GitHub issue firebase/firebase-js-sdk#2366)
+                        */
```

Comment:
Comment should be right above what it refers to. Move it above the `data:` field.